app/views/layouts/_lte_admin_sidebar.html.erb

```diff
@@ -116,6 +116,15 @@
       <p>FAQ</p>
     <% end %>
   </li>
+  
+  <% if Rails.env.production? %>
+    <li class="nav-item <%= 'active' if current_page?(admin_questions_path) %>">
```

Comment:
I think this `if` statement is incorrect?

Reply:
Ahh yeh, I copy and pasted. I think we can just remove it.